simpleLooping/createListOfInteger.py

```diff
@@ -10,3 +10,16 @@
 # Ex:
 #   createListOfInteger(0) # return []
 #   createListOfInteger(4) # return [0, 1, 2, 3]
+from typing import List
+
+def createListOfInteger(N: int) -> List[int]:
+    if N <= 0:
+        return []
+
+    list = []
```

Comment:
integerList I guess? And I don't think Python allow you to use "list" as a variable name anyway.